showcase/app/templates/page-components/form/super-select.hbs

```diff
@@ -1386,38 +1437,48 @@
   <Shw::Text::H3>Content</Shw::Text::H3>
 
   <Shw::Flex as |SF|>
-    <SF.Item @label="No arguments">
+    <SF.Item @label="No text arguments">
```

Comment:
Clarified that there are no _text_ arguments because I had to add in a bunch of required callback functions 😆